src/main/java/org/jenkinsci/plugins/workflow/graphanalysis/FlowChunk.java

```diff
@@ -29,10 +29,10 @@
 import javax.annotation.Nonnull;
 
 /**
- * Common container interface for a series of {@link FlowNode}s with a logical start and end.
- * <p><p> We use this because every plugin has a different way of storing info about the nodes.
+ * <p>Common container interface for a series of {@link FlowNode}s with a logical start and end.</p>
+ * <p>We use this because every plugin has a different way of storing info about the nodes.</p>
```

Comment:
```suggestion
 * <p>Common container interface for a series of {@link FlowNode}s with a logical start and end.
 * <p>We use this because every plugin has a different way of storing info about the nodes.
```